sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountIf.scala

```diff
@@ -34,8 +34,11 @@ import org.apache.spark.sql.types.{AbstractDataType, BooleanType}
   """,
   group = "agg_funcs",
   since = "3.0.0")
-case class CountIf(child: Expression) extends RuntimeReplaceableAggregate
-  with ImplicitCastInputTypes with UnaryLike[Expression] {
+case class CountIf(child: Expression)
+    extends ReplaceableAggregateFunction
+    with RuntimeReplaceable
+    with ImplicitCastInputTypes
+    with UnaryLike[Expression] {
```

Comment:
Ur, is the internal Databricks Scala style guide changed, @HyukjinKwon ? In the community, it's still `2-space` for `extends` and `with`.
- https://github.com/databricks/scala-style-guide

Reply:
oh yeah. my bad :-).